src/main/java/org/rsa/listeners/SlashCommandListener.java

```diff
@@ -15,10 +17,13 @@ public void onSlashCommandInteraction(@NotNull SlashCommandInteractionEvent even
             return;
         }
 
-        try {
-            Commands.getCommand(event.getName()).handleSlashCommand(event);
-        } catch (ValidationException e) {
-            event.reply(e.toEventResponse()).setEphemeral(true).queue();
+        CommandObjectV2 commandObjectV2 = Commands.getCommandV2(event.getName());
+        if (Objects.nonNull(commandObjectV2)) {
```

Comment:
Why not just keep it simple with `commandObject != null`
Objects.nonNull is typically used as a predicate in filter streams.

Reply:
While you are correct in saying nonNull is used as a predicate, I believe functionally it comes down to style preference.

There's no functional different / impact to using `Objects.nonNull` vs `x != null` as behind the scenes, `Objects.nonNull` does exactly that. 😛